build/taskctl/contexts.yaml

```diff
@@ -67,8 +67,8 @@ contexts:
     envfile:
       generate: true
       exclude:
-        - home
-        - path
+        - HOME
+        - PATH
```

Comment:
I think taskctl is still backwards compatible and this is case-insensitive 🤔

Reply:
Yeah it might be but I thought I'd convert it.